lib/httparty/logger/logger.rb

```diff
@@ -3,16 +3,23 @@
 
 module HTTParty
   module Logger
+    def self.formatters
```

Comment:
Actually, I'm ok with formatters. I as I look over how it works I see that these take a logger and format requests and responses so this works.